west.yml

```diff
@@ -193,9 +193,9 @@ manifest:
       groups:
         - hal
     - name: hal_quicklogic
-      revision: b3a66fe6d04d87fd1533a5c8de51d0599fcd08d0
+      revision: c12e7ccb2da626c53f264e7109ff51324fe4c9da
```

Comment:
Please change `revision` to `pull/4/head` so that it references your PR to the QL HAL, and drop `repo-path`/`url` changes.